autoload/neomake.vim

```diff
@@ -2444,9 +2444,16 @@ function! neomake#EchoCurrentError(...) abort
     call neomake#utils#WideMessage(message)
 endfunction
 
-function! neomake#CursorMoved() abort
-    call neomake#EchoCurrentError()
-endfunction
+if !empty(get(g:, 'neomake_cursormoved_func', ''))
+    " this func will be called with one argv ( the current error message)
+    function! neomake#CursorMoved() abort
+        call call(g:neomake_cursormoved_func, [neomake#GetCurrentErrorMsg()])
+    endfunction
+else
+    function! neomake#CursorMoved() abort
+        call neomake#EchoCurrentError()
+    endfunction
+endif
```

Comment:
I think it would be better to provide an API to get info about the current error (and errors in general then maybe), so that you can just have your own autocommand?!

Reply:
yeah, what I need is just a plugin API, now `neomake#GetCurrentErrorMsg()` is what I want, and I hope to disable the default autocmd for cursormoved.

Reply:
then maybe the following will work already?

```vim
au! neomake CursorMoved
au! neomake VimResized
```

I agree though that e.g. a separate augroup (`neomake_current_error`) would be easier / better.